### PR TITLE
Fix sending kodi notifications using python3

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

fix for #6398

### CODE REVIEW

File: medusa/notifiers/kodi.py [hunk 1]
```diff
@@ -281,7 +283,7 @@ def clean_library(self):
         # If no errors, return True. Otherwise keep sending command until all hosts are cleaned
         return clean_library
 
-    def _update_library(self, host=None, series_name=None):
+    def _update_library(self, host=None, series_name=None):  # pylint: disable=too-many-return-statements, too-many-branches
```
Comment (medariox) [inline medusa/notifiers/kodi.py]: Why the pylint?

Comment (p0psicles) [reply medusa/notifiers/kodi.py]: Not sure. Didn't put it in. Maybe because of a merge/rebase? I'll remove it

**Comment by p0psicles** [comment]
Tested a test notification. But I think It was put there for when show names with unicode chars in are send. Need to replicate that

**Comment by p0psicles** [comment]
Tested using this command. `result = self._notify_kodi('Мажор', 'Мажор - S02E12 - Серия 24')`, seemed to work okay.

**Comment by medariox** [comment]
DeepCode retry

**Pull request merged**